scripts/docker/centos.7/Dockerfile

```diff
@@ -23,7 +23,8 @@ RUN yum -q -y install unzip libunwind gettext libcurl-devel openssl-devel zlib l
 
 # Install Build Prereqs
 # CMake 3.3.2 from GhettoForge; LLVM 3.6.2 built from source ourselves;
```

Comment:
I'm pretty sure the comment's still true: the CMake here is a mirror of the package built by GhettoForge (that's the `gf`), and the other packages are ones that we built from source ourselves.